javascript/extractor/src/com/semmle/js/extractor/AutoBuild.java

```diff
@@ -666,6 +666,9 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs)
               throws IOException {
             if (!dir.equals(currentRoot[0]) && (excludes.contains(dir) || dir.toFile().isHidden()))
               return FileVisitResult.SKIP_SUBTREE;
+            if (dir.resolve("codeql-database.yml").toFile().exists()) {
```

Comment:
Have we checked on `#codeql-cli` if that is the only new thing we should ignore?

Reply:
This ought to suffice for excluding databases made by the new CLI.

It won't take if you trick Odasa into creating a snapshot in a subdirectory of the source tree, but I believe doing so would takes quite a bit more coercion.

Reply:
Perhaps `qlpack.yml`, to avoid extracting test cases in existing qlpacks?

Reply:
That would be problematic for extracting a test directory that has its own `qlpack.yml`.

Reply:
By the way, why `...toFile().exists()` rather than `Files.exists(...)`?

Reply:
Old habit I suppose. Switching to `Files.exists()`.